3rdParty/LightPcapNg/LightPcapNg/include/light_pcapng_ext.h

```diff
@@ -34,6 +34,7 @@ extern "C" {
 #include <stdint.h>
 #ifdef _MSC_VER
 #include <Winsock2.h>
+#include <time.h>
 #else
 #include <sys/time.h>

```

Comment:
I think yes, as it contains definition of `timespec`

Reply:
I think that `sys/time.h` contains only the definition for `timeval` whereas `timespec` is defined in `time.h`, please take a look [here](https://pubs.opengroup.org/onlinepubs/009695399/basedefs/sys/time.h.html) and [here](https://en.cppreference.com/w/c/chrono/timespec). 

Since `timeval` is now totally removed from this file I'm not sure `sys/time.h` is still needed. Would you mind testing that?

Reply:
I tried removing it and compilation failed. You were right 😃